packages/features/bookings/lib/handleNewBooking/scheduleNoShowTriggers.ts

```diff
@@ -7,28 +10,57 @@ type ScheduleNoShowTriggersArgs = {
   booking: {
     startTime: Date;
     id: number;
+    location: string | null;
   };
   triggerForUser?: number | true | null;
   organizerUser: { id: number | null };
   eventTypeId: number | null;
   teamId?: number | null;
   orgId?: number | null;
+  destinationCalendars?: DestinationCalendar[] | null;
 };
 
 export const scheduleNoShowTriggers = async (args: ScheduleNoShowTriggersArgs) => {
-  const { booking, triggerForUser, organizerUser, eventTypeId, teamId, orgId } = args;
+  const { booking, triggerForUser, organizerUser, eventTypeId, teamId, orgId, destinationCalendars } = args;
+
+  const isDailyVideoLocation = booking.location === DailyLocationType || booking.location?.trim() === "";
+  const isGoogleMeetLocation = booking.location === "integrations:google:meet";
+
+  if (!isGoogleMeetLocation && !isDailyVideoLocation) return;
 
-  // Add task for automatic no show in cal video
   const noShowPromises: Promise<any>[] = [];
 
-  const subscribersHostsNoShowStarted = await getWebhooks({
+  const hostNoShowTriggerEvent = isDailyVideoLocation
```

Comment:
My previous comment applies here. This list will continue to grow larger and larger for the more video integrations we detect no shows for. Do we absolutely have to do this?